### PR TITLE
Enable async backing on Kusama

### PR DESCRIPTION
**Pull request by Sophia-Gold**
This enables async backing in the Kusama runtime. Note that for candidates to actually be backed asynchronously, a subsequent governance proposal changing two new parameters is needed.

### CODE REVIEW

File: relay/kusama/src/lib.rs [hunk 1]
```diff
@@ -2040,6 +2041,18 @@ sp_api::impl_runtime_apis! {
 				key_ownership_proof,
 			)
 		}
+
+			fn minimum_backing_votes() -> u32 {
+			parachains_runtime_api_impl::minimum_backing_votes::<Runtime>()
```
Comment (eskimor) [inline relay/kusama/src/lib.rs:2046]: Formatting seems off.

**Review by eskimor — APPROVED** [review]
Looks good, although I think we should now also bump the version of the trait to (7) - at least once we have this on Polkadot as well. @tdimitrov ?

File: relay/kusama/src/lib.rs (hunk 1 above)
Comment (Sophia-Gold) [reply relay/kusama/src/lib.rs:2046]: Done in 9f5cef432e41972579d394c66cee74d8da2043e1. Not sure why this wasn't caught by rustfmt.

File: relay/kusama/src/lib.rs [hunk 2]
```diff
@@ -1910,6 +1910,7 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 
+		#[api_version(7)]
```
Comment (tdimitrov) [inline relay/kusama/src/lib.rs:1913]:
```suggestion
	#[api_version(7)]
```

**Comment by tdimitrov** [comment]
> Looks good, although I think we should now also bump the version of the trait to (7) - at least once we have this on Polkadot as well. @tdimitrov ?

Yes, but it is already done?
https://github.com/polkadot-fellows/runtimes/pull/90/files#diff-efa4caeb17487ecb13d8f5eb7863c3241d84afa2e73fbf25909a2ca89df0f362R1913

**Comment by Sophia-Gold** [comment]
> Looks good, although I think we should now also bump the version of the trait to (7) - at least once we have this on Polkadot as well. @tdimitrov ?

I assume you mean 8?

**Comment by eskimor** [comment]
> Yes, but it is already done?

No. I meant the trait itself, but this can only be done once we also launched on Polkadot: So disregard.

**Review by ordian — APPROVED** [review]
Expect chaos.

**Review by joepetrowski — COMMENTED** [review]
How about enabling system parachains to use it?

**Comment by Sophia-Gold** [comment]
> How about enabling system parachains to use it?

They won't be able to make use of it until we propose an update to the new parameters after this is enacted. 

Glutton is already upgraded in Polkadot-SDK. I'll move that here. Can your team upgrade the others?

**Comment by joepetrowski** [comment]
> Glutton is already upgraded in Polkadot-SDK. I'll move that here. Can your team upgrade the others?

Yes

**Pull request closed without merging**